### ISSUE TITLE
Read CompletionItemKind for suggestions 

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#59579]
Testing #40597

**Repo**
1. Using VoiceOver on a mac
1. Trigger suggestions in a js file
1. use arrow keys to navigate suggestions

**Feature request**
Currently all suggestions are read as `LABEL, suggestion`:

<img width="526" alt="screen shot 2018-09-27 at 1 46 01 pm" src="https://user-images.githubusercontent.com/12821956/46173641-ba682500-c25b-11e8-9f66-a8cdfed12caf.png">

I think we should read the suggestion kind as well, perhaps: `LABEL, snippet` or `LABEL, method`

**Comment by isidorn** [comment]
This feels like low hanging fruit. Though I think this would actually be too noisy.
@neurrone would you find the kind read out to be useful?

**Comment by jrieken** [comment]
@pawelurbanski and I were - before we would say "suggestion" or "snippet suggestion" and that was perceived as noisy. That's why we only read the label. Anyways, open for feedback, reading out the completion type (snippet, function, variable etc) is easy to add but it should only be done if it helps

**Comment by Neurrone** [comment]
@isidorn having the type available could be useful. Having it read after the label has no risk of being too noisy, since the suggestion itself would be read first. 

In terms of whether it should be announced before or after the suggestion itself, I'm not sure which might be more useful. I recommend announcing it after first, and only changing it if there's demand for it.

**Comment by isidorn** [comment]
I would only announce it after to reduce noise.
@jrieken maybe we change this and try to get some feedback in insiders. What do you think?

**Comment by jrieken** [comment]
Waiting for @pawelurbanski to provide his feedback

**Comment by pawelurbanski** [comment]
Providing the type after is definitely an option. Speaking of options: Making it a setting would be just perfect.

The problem about such metadata comes from the fact that every language as its naming notation. Sometimes it makes the type easily understood purely based on the name.
My idea for a format of the alert is as follows:
Item: Type – the “:” is silent – at least I keep my verbosity settings to some punctuation characters, so the screen reader will read it fast.

**Comment by isidorn** [comment]
If we introduce a setting I would like except some general accessibility setting. Like
`accessibility.verbosity`, I would not have a setting only for the completion item kind specifically.

Also `item: type` makes sense to me.

**Comment by isidorn** [comment]
Closing as out of scope for now. We would not change this, and being concise makes sense based on user feedback.
